GridTariffAPI.lib/Services/TariffQueryService.cs

```diff
@@ -360,9 +360,9 @@ private async Task<SeasonDataAccumulator> ProcessSeasonAsync(TariffPrice tariffP
 
         public List<int> GetDistinctFixedPriceMonths(DateTimeOffset fromDateUtc, DateTimeOffset toDateUtc)
         {
-            var fromDateLocaled = _serviceHelper.ToConfiguredTimeZone(fromDateUtc.DateTime);
+            var fromDateLocaled = _serviceHelper.ToConfiguredTimeZone(fromDateUtc.UtcDateTime);
             fromDateLocaled = fromDateLocaled.AddDays(1 - fromDateLocaled.Day);
-            var toDateLocaled = _serviceHelper.ToConfiguredTimeZone(toDateUtc.DateTime);
+            var toDateLocaled = _serviceHelper.ToConfiguredTimeZone(toDateUtc);
```

Comment:
Kan ikke ToConfiguredTimeZone(fromDateUtc) benyttes direkte (uten .UtcDateTime)? Og håndtering av fromDateUtc og toDateUtc bør være like.

Reply:
Enig, det er ikke behov for .UtcDateTime.